### PR TITLE
Add option to preserve formatting in additional comments

### PR DESCRIPTION
**Pull request by kageiit**

### CODE REVIEW

**Comment by kageiit** [comment]
Resolves #75

**Comment by jjx** [comment]
@sectioneight, thoughts?

**Comment by sectioneight** [comment]
Looks good. I have a branch where I tried to make this checkbox conditionally displayed based on whether commenting was enabled. We can merge this as is though. A change log entry would be good however (I can do it later if you just want to merge this)

File: src/main/java/com/uber/jenkins/phabricator/CommentBuilder.java [hunk 1]
```diff
@@ -123,7 +125,11 @@ public void addUserComment(String customComment) {
         if (hasComment()) {
             comment.append("\n\n");
         }
-        comment.append(String.format("```\n%s\n```\n\n", customComment));
+        if (preserveFormatting) {
+            comment.append(String.format("\n%s\n", customComment));
```
Comment (sectioneight) [inline src/main/java/com/uber/jenkins/phabricator/CommentBuilder.java]: I think the leading new line may be excessive here. The one below is to separate from the opening back ticks

**Comment by kageiit** [comment]
@sectioneight updated changelog and removed extra '\n'

**Comment by sectioneight** [comment]
ship.it.

I will figure out how to make the checkboxes collapse in a future pr

**Pull request merged**